core-services/image-mirroring/integr8ly/mapping_integr8ly_operator_3scale-next-0_7_0

```diff
@@ -0,0 +1,2 @@
+registry.svc.ci.openshift.org/integr8ly/3scale-next-0.7.0:integreatly-operator quay.io/integreatly/integreatly-operator:3scale-next-0.7.0
+registry.svc.ci.openshift.org/integr8ly/3scale-next-0.7.0:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:3scale-next-0.7.0
```

Comment:
You will need to add a new line here for the rhoam test harness image https://github.com/openshift/release/blob/master/core-services/image-mirroring/integr8ly/mapping_integr8ly_operator#L3. The delorean command you used to generate this should be updated as well ideally.

```registry.svc.ci.openshift.org/integr8ly/i3scale-next-0.7.0:integreatly-operator-test-harness-osde2e quay.io/integreatly/integreatly-operator-test-harness:osde2e-3scale-next-0.7.0 ```

Reply:
update: eventually we will "merge" osde2e and functional test images into a single testing image that can be used for testing using osde2e framework or nightly jenkins pipelines